src/main/java/net/cardosi/mojo/cache/CachedProject.java

```diff
@@ -466,9 +470,12 @@ public void run() {
 
         Preconditions.checkArgument(config.getDependencyMode() == DependencyOptions.DependencyMode.SORT_ONLY, "With compilationLevel=" + BUNDLE_JAR + " only dependencyMode=SORT_ONLY is supported");
 
+        LanguageMode languageOut = LanguageMode.valueOf(config.getLanguageOut());
+
         boolean success = closureCompiler.compile(
                 CompilationLevel.BUNDLE,
                 DependencyOptions.DependencyMode.SORT_ONLY,
+                languageOut,
```

Comment:
Like below, where STABLE_OUT was specified and hardcoded, this will have no effect - BUNDLE mode just spits out the same code it reads in, reordered and wrapped so that it works in a single file. Removed in the version that I'm merging, replaced with NO_TRANSPILE, but i'm almost certain that these values don't do anything at all anyway.